src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -660,7 +660,7 @@ private IEnumerable<ISymbol> GetSymbolsInNewSolution(Document newDocument, Seman
                     foreach (var language in documentsFromAffectedProjects.Select(d => d.Project.Language).Distinct())
                     {
                         solution.Workspace.Services.GetLanguageServices(language).GetService<IRenameRewriterLanguageService>()
-                            .TryAddPossibleNameConflicts(symbol, _replacementText, _possibleNameConflicts);
+                            ?.TryAddPossibleNameConflicts(symbol, _replacementText, _possibleNameConflicts);
```

Comment:
this seems somehwat unnecessary.  assuming, of course that GEtDocumentsByRename only returns actual C#/VB docs...

Reply:
So hence my desire for discussion: right now it was bringing in F# documents too. And perhaps F# *should* be able to extend this point by defining this interface?

Reply:
Oh interesting.... i'm suprrised that documentsFromAffectedProjects was pulling in F# docs in the first place.  I would have thought that it only pulled in C#/VB docs.  

--

That said, i agree that it woudl be interesting to allow F# to play in this space since it is a .net language.  so i'm ok with your fix here now :)

Reply:
This looks reasonable to me since F# doesn't implement `IRenameRewriterLanguageService`. At some point we might, but not anytime soon.